src/internal/m365/service/groups/export.go

```diff
@@ -134,3 +148,39 @@ func (h *baseGroupsHandler) ProduceExportCollections(
 
 	return ec, el.Failure()
 }
+
+// ========================================================================== //
+//                              groupsHandler
+// ========================================================================== //
+
+// groupsHandler contains logic for handling data and performing operations
+// (e.x. restore) regardless of whether they require contact with external M365
+// services or not.
+type groupsHandler struct {
+	baseGroupsHandler
+	apiClient      api.Client
+	resourceClient idname.GetResourceIDAndNamer
+}
+
+func (h *groupsHandler) IsServiceEnabled(
+	ctx context.Context,
+	resource string,
+) (bool, error) {
+	// TODO(ashmrtn): Move free function implementation to this function.
```

Comment:
Question: what is `free function` in this context?

Reply:
ah, `IsServiceEnabled()` in this case. Right now it's still setup to be called from something more generic. Since we're now working to centralize things in the service level handler it seems like it would be good to move that here too

In general "free function" is a function that's not on a struct/class (linking [SO](https://stackoverflow.com/a/4863328) cause I don't want to type a full code example)

Reply:
Huh, interesting.  I've always referred to that as the difference between `methods` (not free) and `functions` (free).

Reply:
yea, I think that's also terminology that's used. I'm honestly kind of terrible at remembering that though and tend to use "methods" and "functions" interchangeably 😅

Reply:
hah, you're not alone